pkg/ddc/alluxio/shutdown.go

```diff
@@ -170,24 +171,20 @@ func (e *AlluxioEngine) destroyWorkers(workers int32) (err error) {
 		labelExclusiveName = common.Exclusive
 	)
 
-	err = e.List(context.TODO(), nodeList, &client.ListOptions{})
+	labelNames := []string{labelName, labelTotalname, labelDiskName, labelMemoryName, labelCommonName}
+	datasetLabels, err := labels.Parse(fmt.Sprintf("%s=true", labelCommonName))
 	if err != nil {
 		return
 	}
 
-	labelNames := []string{labelName, labelTotalname, labelDiskName, labelMemoryName, labelCommonName}
-
-	runtimeInfo, err := e.getRuntimeInfo()
+	err = e.List(context.TODO(), nodeList, &client.ListOptions{
```

Comment:
I suggest that we should only use `labelCommonName` to search, because labelDiskName and labelMemoryName doesn't exist for sure.

Reply:
You mean node search by nodeselecctor via `labelCommonName`? This is how it is done at the moment。: )

Reply:
Sorry， you are correct.